bullet_train/docs/field-partials/file-field.md

```diff
@@ -8,8 +8,6 @@ In addition, Bullet Train has integrated the direct-uploads feature of Active St
 
 ## Example
 
-Add a 'document' file as an attachment to a `Post` model:
```

Comment:
I think the purpose of this line was to describe in a general way what the example is about, but I don't think it was worded all that well. I propose instead of removing it that we change it to something like:

The following steps illustrate how to add a `document` file attachment to a `Post` model.

Reply:
Yes, I think saying "The following steps illustrate..." which is descriptive as opposed to saying "Add this" will be easier in reading the flow of the docs here. Adding now